src/definitions/collections/menu.less

```diff
@@ -84,7 +84,7 @@
   -webkit-tap-highlight-color: transparent;
   flex: 0 0 auto;
   user-select: none;
-
+  cursor: pointer;
```

Comment:
As @donaggio pointed out, i also don't think it is correct to __always__ set the cursor to pointer to __any__ kind of menu item (not only tabs would be affected)

So what about adding the `cursor:pointer` only to `.ui.tabular.menu .item` instead?

Reply:
I think that menu items should always set the cursor to `pointer` except `.disabled` item, so that end-users can recognize items are clickable, in either case `a.item` or `div.item`.

As a SUI/FUI user, it is not clear for me that `div.item` requires `.link` class to be link-ish while `a.item` requires no `.link` even when no `href` given.
I even don't known there is `.link.item` until today.
I think this is a sign of confusing/complex point of SUI/FUI and should be simplified.

Reply:
ok, convinced. 😄 
Btw the `link` is still necessary to provide hovering functionality to change color/background on a `did` tag, so completely remove `.link` wouldn't be an option (as @donaggio propably guessed)

Reply:
I'm not convinced yet, don't get me wrong I complete agree with your points and I feel that way too about how confusing it is when you create a menu then the items don't seem clickable I just want to try and see if there was any comments made by Jack about this before we move forward.

Reply:
Menu items can contain just about anything. This can be really useful sometimes, especially with vertical menus: I know I used it to put an entire mini-form in it one time, and I certainly didn't want the mouse cursor to appear as if the whole form was a link.

Reply:
@donaggio Agreed, I can't find any comments on the issue from Jack but your point is enough for me to deny this change this is a good example https://fomantic-ui.com/collections/menu.html#secondary-menu of what your saying.

Reply:
@donaggio
That makes sense to keep div.item as simple container without pointer cursor.
I  think it is better to close this PR/issue as wontfix.
There are some use cases and some solutions (link item or a.item without href).

Suggestions are welcome.

Reply:
So we don't even apply the pointer to the tabular menu item only then? , the original intention of the related issue)

Reply:
No, we don't apply this PR to tabular menu.
There is tabular menu example of docs having search input in item.

Reply:
Yeah we can do that would make sense.